securedrop/manage.py

```diff
@@ -42,7 +42,7 @@ def reset(args):
     """
     # Erase the development db file
     if not hasattr(config, 'DATABASE_FILE'):
-        raise Exception("TODO: ./manage.py doesn't know how to clear the db "
```

Comment:
everyone is using sqlite and they should just be warned than non-sqlite is not supported, so this is a reasonable behavior change imho